packages/services/api/src/modules/collection/resolvers/DocumentCollection.ts

```diff
@@ -0,0 +1,13 @@
+import { CollectionProvider } from '../providers/collection.provider';
+import type { DocumentCollectionResolvers } from './../../../__generated__/types.next';
+
+export const DocumentCollection: DocumentCollectionResolvers = {
+  id: root => root.id,
+  name: root => root.title,
+  description: root => root.description,
+  operations: (root, args, { injector }) =>
+    injector.get(CollectionProvider).getOperations(root.id, args.first, args.after),
+  createdBy: async (_parent, _arg, _ctx) => {
+    /* DocumentCollection.createdBy resolver is required because DocumentCollection.createdBy exists but DocumentCollectionMapper.createdBy does not */
+  },
```

Comment:
This was never implemented in this module, and I couldn't find this being implemented somewhere else... But it's not erroring at runtime... 
So, the only reason I can think of is this field is not being queried in any client. 

I'll need advice on how to turn the returned `createdByUserId` to a `User` object here 🙏 
Otherwise, we can ignore it in this migration with TS override and come back later

Reply:
You could call `injector.get(Storage).getUserById` or remove it altogether since it is not used from the frontend.

Reply:
Yeah, I think it can be removed also from the schema

Reply:
This is now done